core/py/src/braintrust_core/score.py

```diff
@@ -51,8 +51,12 @@ async def _run_eval_async(self, output, expected=None, **kwargs) -> Score:
         # By default we just run the sync version in a thread
         return self._run_eval_sync(output, expected, **kwargs)
 
+    @classmethod
```

Comment:
Out of curiosity, why do we need our own method for this? Is it an expected method for some caller (if so maybe worth documenting).

Reply:
I think I can revert this since I reworked how class names are defined